src/main/java/com/base2services/jenkins/SqsQueueHandler.java

```diff
@@ -10,6 +10,7 @@
 import hudson.util.SequentialExecutionQueue;
 import hudson.util.TimeUnit2;
 
+import java.util.ArrayList;
```

Comment:
Reviewer note: 
Changes to this file are unrelated to the new feature. They are improvements to the existing code that makes troubleshooting easier, especially AWS permissions issues.